### DISCUSSION TITLE
HomeAssistant TasmoAdmin

### DISCUSSION THREAD

**Discussion by lodv** [community/community#22831] [New to GitHub]
<p>Is it possible to save the username and Password for TasmoAdmin in HomeAssistant in the file editor?<br>
Where can I find the code for this?<br>
Thank you in advance.<br>
Yvonne</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>It looks like you’re asking a question about TasmoAdmin that has a GitHub repository: <a href="https://github.com/hassio-addons/addon-tasmoadmin">https://github.com/hassio-addons/addon-tasmoadmin</a></p>
<p>If you’re asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:</p>
<ol>
<li>Check the README for instructions on how to operate the software, pointers to documentation or troubleshooting info</li>
<li>Check the <a href="https://help.github.com/articles/adding-support-resources-to-your-project/">SUPPORT file</a>, if one exists</li>
<li>Check the <a href="https://help.github.com/articles/setting-guidelines-for-repository-contributors/">CONTRIBUTING guide</a>, if one exists</li>
</ol>
<p>All of these documents can be found, if they exist, in the repository where you found the software itself.</p>
<p>Additionally, I was able to find this description of TasmoAdmin that includes instructions on how to get support for it: <a href="https://community.home-assistant.io/t/home-assistant-community-add-on-tasmoadmin/54155">https://community.home-assistant.io/t/home-assistant-community-add-on-tasmoadmin/54155</a></p>
<p>I hope that helps!</p>

**Comment by lodv** [comment]
<p>Thank you. I will.<br>
Yvonne</p>
